packages/dataviews/src/dataforms-layouts/index.tsx

```diff
@@ -18,3 +19,22 @@ const FORM_LAYOUTS = [
 export function getFormLayout( type: string ) {
 	return FORM_LAYOUTS.find( ( layout ) => layout.type === type );
 }
+
+const FORM_FIELD_LAYOUTS = [
```

Comment:
Don't we have a `FORM_LAYOUT` constant above? Why do we need both?

Reply:
That is still referring to the current form layout that determines it for the entire form.
This can be removed if we go ahead with the form field layout instead.